libraries/botbuilder-middleware-watson-nlu/example/app-watson.js

```diff
@@ -0,0 +1,31 @@
+const { BotFrameworkAdapter } = require("botbuilder");
+const restify = require("restify");
+const { EmotionDetection } = require("../lib/index");
+
+require("dotenv").config();
+
+const server = restify.createServer();
+server.listen(process.env.port || process.env.PORT || 3978, function () {
+    console.log(`${server.name} listening to ${server.url}`);
+});
+
+const adapter = new BotFrameworkAdapter({ 
+    appId: process.env.MICROSOFT_APP_ID, 
+    appPassword: process.env.MICROSOFT_APP_PASSWORD 
+});
+
+const opts = {
+    engine: 1
```

Comment:
Where does this refer to? I can't find this enum.

Reply:
I need to remove this. It's no longer valid.